packages/flutter/lib/src/material/autocomplete.dart

```diff
@@ -193,6 +194,12 @@ class Autocomplete<T extends Object> extends StatelessWidget {
   /// default.
   final AutocompleteOptionsViewBuilder<T>? optionsViewBuilder;
 
+  /// defines the maximum height the options can take
+  /// if optionsViewBuilder is null
+  ///
+  /// defaults to 200.0
```

Comment:
Docs should always start with upper case letter.

Reply:
thanks @pedromassango  by revert do you mean to revert the commit?
And is there any cli tool we can use to format the code  or it has to be done manually seems like we do not use ```dartfmt``` here ?

Reply:
I would suggest to just go over all the changes you've made and revert each unwanted change, that iw what I've been doing  back in the time (until I disabled auto-format).

Yes, the Flutter source code does not use dartfmt tool. I'm not aware of ny tool that could help with this. Let me know if you find any, I would like to use it as well :).

Reply:
Thanks but can you point me to a doc(if any) where I could know what changes to revert (e.g you specified the upper case letter or extra space)?
or are these rules bases on the [Effective Dart](https://dart.dev/guides/language/effective-dart)

Reply:
And one more thing is there a way we could run all the tests locally before pushing it here.

Reply:
It seems fixed now